Running make in a clean checkout of Volto crashes

This is the entire log. The commands I did:

```
git clone ...
cd clone ...
make
```


```
 ◰³ base  ~/work   …  
 git clone git@github.com:plone/volto.git volto-try                                                                                                                                                                    (base)  Thu 16 Dec 2021 11:39:40 AM EET
Cloning into 'volto-try'...
remote: Enumerating objects: 62534, done.
remote: Counting objects: 100% (24/24), done.
remote: Compressing objects: 100% (22/22), done.
remote: Total 62534 (delta 2), reused 9 (delta 1), pack-reused 62510
Receiving objects: 100% (62534/62534), 71.63 MiB | 17.56 MiB/s, done.
Resolving deltas: 100% (39555/39555), done.
 ◰³ base  ~/work   …  
 cd volto-try/                                                                                                                                                                                                   (base) 6.6s  Thu 16 Dec 2021 11:39:56 AM EET
 ◰³ base  ~/w/volto-try    
 make                                                                                                                                                                                                                  (base)  Thu 16 Dec 2021 11:40:00 AM EET
+ make build-backend
make[1]: Entering directory '/home/tibi/work/volto-try'
+ make -C ./api/ build
make[2]: Entering directory '/home/tibi/work/volto-try/api'
+ python3 -m venv .
+ bin/pip install --upgrade pip
Requirement already satisfied: pip in ./lib/python3.9/site-packages (21.1.1)
Collecting pip
  Using cached pip-21.3.1-py3-none-any.whl (1.7 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 21.1.1
    Uninstalling pip-21.1.1:
      Successfully uninstalled pip-21.1.1
Successfully installed pip-21.3.1
+ bin/pip install --upgrade wheel
Collecting wheel
  Using cached wheel-0.37.0-py2.py3-none-any.whl (35 kB)
Installing collected packages: wheel
Successfully installed wheel-0.37.0
+ bin/pip install -r requirements.txt
Collecting setuptools==42.0.2
  Using cached setuptools-42.0.2-py2.py3-none-any.whl (583 kB)
Collecting zc.buildout==2.13.3
  Using cached zc.buildout-2.13.3-py2.py3-none-any.whl (153 kB)
Collecting requests
  Using cached requests-2.26.0-py2.py3-none-any.whl (62 kB)
Collecting charset-normalizer~=2.0.0
  Downloading charset_normalizer-2.0.9-py3-none-any.whl (39 kB)
Collecting idna<4,>=2.5
  Using cached idna-3.3-py3-none-any.whl (61 kB)
Collecting urllib3<1.27,>=1.21.1
  Using cached urllib3-1.26.7-py2.py3-none-any.whl (138 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2021.10.8-py2.py3-none-any.whl (149 kB)
Installing collected packages: urllib3, setuptools, idna, charset-normalizer, certifi, zc.buildout, requests
  Attempting uninstall: setuptools
    Found existing installation: setuptools 56.0.0
    Uninstalling setuptools-56.0.0:
      Successfully uninstalled setuptools-56.0.0
Successfully installed certifi-2021.10.8 charset-normalizer-2.0.9 idna-3.3 requests-2.26.0 setuptools-42.0.2 urllib3-1.26.7 zc.buildout-2.13.3
+ bin/buildout
Creating directory '/home/tibi/work/volto-try/api/eggs'.
Getting distribution for 'mr.developer==2.0.1'.
Got mr.developer 2.0.1.
Getting distribution for 'six==1.16.0'.
Got six 1.16.0.
mr.developer: Creating missing sources dir /home/tibi/work/volto-try/api/src.
mr.developer: Queued 'plone.volto' for checkout.
mr.developer: Cloned 'plone.volto' with git using branch 'coresandboxtests' from 'https://github.com/plone/plone.volto.git'.
mr.developer: git cloning of 'plone.volto' failed.
mr.developer: fatal: Remote branch coresandboxtests not found in upstream origin
mr.developer: 
mr.developer: There have been errors, see messages above.
make[2]: *** [Makefile:40: build] Error 1
make[2]: Leaving directory '/home/tibi/work/volto-try/api'
make[1]: *** [Makefile:53: build-backend] Error 2
make[1]: Leaving directory '/home/tibi/work/volto-try'
make: *** [Makefile:44: build] Error 2
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,7 +4,7 @@
 
 ### Breaking
 
-- Move `theme.js` import to top of the client code, so it take precedence over any other inline imported CSS. This is not an strict breaking change, but it's worth to mention it as might be important and kept in mind.  @sneridagh
+- Move `theme.js` import to top of the client code, so it take precedence over any other inline imported CSS. This is not an strict breaking change, but it's worth to mention it as might be important and kept in mind. @sneridagh
 
 ### Feature
 
@@ -20,6 +20,7 @@
 
 - Upgrade to react 17.0.2 @nzambello
 - Upgrade caniuse-lite 1.0.30001286 @tiberiuichim
+- fix:correctly checkout plone.volto in buildout @nileshgulia1
 
 ## 14.0.0-alpha.42 (2021-12-13)
 
@@ -45,7 +46,7 @@
   and develop addons (in `src/addons` folder).
 - Add new listing block option "fullobjects" per variation @ksuess
 - Style checkboxes @nileshgulia1
-- Allow loading .less files also from a Volto project's `src` folder.  @tiberiuichim
+- Allow loading .less files also from a Volto project's `src` folder. @tiberiuichim
 
 ### Bugfix
 

--- a/api/buildout.cfg
+++ b/api/buildout.cfg
@@ -8,11 +8,12 @@ versions = versions
 extensions = mr.developer
 auto-checkout = plone.volto
 always-checkout = force
+show-picked-versions = true
 
 [sources]
-plone.volto = git https://github.com/plone/plone.volto.git branch=coresandboxtests
+plone.volto = git https://github.com/plone/plone.volto.git branch=main
 plone.rest = git git@github.com:plone/plone.rest.git branch=master
-plone.restapi = git git://github.com/plone/plone.restapi.git pushurl=git@github.com:plone/plone.restapi.git branch=improvesupportformissingvaluedefaultstory
+plone.restapi = git git://github.com/plone/plone.restapi.git pushurl=git@github.com:plone/plone.restapi.git branch=master
 
 [instance]
 recipe = plone.recipe.zope2instance
@@ -72,10 +73,13 @@ eggs = ${instance:eggs}
 [versions]
 setuptools =
 zc.buildout =
-plone.restapi = 8.16.2 # force to latest p.restapi
-plone.rest = 2.0.0a1 # new JSON only traversal
+plone.restapi = 8.13.0
+# force to latest p.restapi
+plone.rest = 2.0.0a1 
+# new JSON only traversal
 plone.app.iterate = 4.0.2 # Using working copy support in Volto requires it
-plone.app.vocabularies = 4.3.0 # plone.volto requires it
+plone.app.vocabularies = 4.3.0
+# plone.volto requires it
 
 plone.volto =
 robotframework =

